tools/pagerduty-migrator/scripts/README.md

```diff
@@ -0,0 +1,9 @@
+# Grafana Oncall Scripts
```

Comment:
Great idea to make a user migrator from PD to Grafana! I propose to make this as a separate PR though.

Reply:
I just created the PR: https://github.com/grafana/oncall/pull/403